public/js/logix.js

```diff
@@ -1,1393 +1,1393 @@
-// Size of canvas
```

Comment:
I need to check why GitHub is having trouble showing this diff...all I did was add the extra component in the list.

Reply:
looks like it was a whitespace change in all lines. I've reverted them.